RecoMuon/MuonSeedGenerator/src/MuonSeedPtExtractor.cc

```diff
@@ -391,6 +391,16 @@ int MuonSeedPtExtractor::stationCode(MuonTransientTrackingRecHit::ConstMuonRecHi
     if(result == 1 && (1 == cscID.ring() ||  4 == cscID.ring()) )
        result = 0;
   }
+  else if( hit->isGEM() ){
+    GEMDetId gemID(detId);
+    //std::cout<<"first (GEM) E/S/R/C = "<<gemID.endcap()<<"/"<<gemID.station()<<"/"<<gemID.ring()<<"/"<<gemID.chamber()<<std::endl;
+    result = gemID.station();
```

Comment:
... actually, if you will be reusing settings for CSC, shouldn't both ME0 and GE11 be set to ```result = 0``` as in the CSC case?